Glyssen/Dialogs/VoiceActorAssignmentDlg.cs

```diff
@@ -39,9 +39,9 @@ public VoiceActorAssignmentDlg(Project project)
 
 				foreach (KeyValuePair<int, CharacterGroup> pair in charGroupTemplate.CharacterGroups)
 					characterGroups.Add(pair.Value);
-			}
 
-			m_project.CharacterGroupList.PopulateEstimatedHours(m_project.IncludedBooks);
+				m_project.CharacterGroupList.PopulateEstimatedHours(m_project.IncludedBooks);
```

Comment:
Estimated hours are stored in characterGroups.xml along with other group information. This makes sense because we are also storing characterIds. When I tested it, it loaded correctly.